src/noise.ts

```diff
@@ -49,10 +49,18 @@ export class Noise implements INoiseConnection {
     this.useNoisePipes = false
 
     if (staticNoiseKey) {
-      const publicKey = x25519.publicKeyCreate(staticNoiseKey)
+      const keyPair = x25519.generateKeyPairFromSeed(staticNoiseKey)
```

Comment:
this seems wrong, given key is privateKey not seed imo

Reply:
Its called generateKeyPairFromSeed but it actually accepts SECRET_KEY_LENGTH of 32 which is not seed but actually the 'staticNoiseKey'. 
Maybe I misread something?

Reply:
you are right, weird naming. Just add code comment as to not confuse other people